server/shared/src/test/scala/org/http4s/server/middleware/CORSSuite.scala

```diff
@@ -125,8 +125,8 @@ class CORSSuite extends Http4sSuite {
     val req = buildRequest("/bar").withHeaders("Origin" -> "http://blah.com/")
     cors2
       .orNotFound(req)
-      .map(resp => resp.status.code == 403)
-      .assert
+      .map(_.status)
+      .assertEquals(Status.Forbidden)
```

Comment:
What was this about?

Reply:
That's a test I found failing, and I don't know what status it's actually returning, and I can't make it fail locally.

Reply:
Humph, really weird. Maybe some weird numerics thing (all numbers in JS are represented as `Double` so...). I'll try and have a closer look at it later.

Reply:
Oh, unless I misunderstood, you're not sure if this change actually makes the test pass?

Reply:
As it was, it was passing on some builds and failing on others, including on a JVM, and I don't have any idea WTF is going on.

Reply:
Something stinks in the output.

```
org.http4s.server.middleware.CORSSuite:
```

with no successes (`+`) or failues (`==> x`) reported under it.  And the only other mention is 

```
[error] Failed: Total 253, Failed 1, Errors 0, Passed 252
[error] Failed tests:
[error] 	org.http4s.server.middleware.CORSSuite
```

I wonder whether `ErrorReporting` is doing something wicked.

Reply:
How are you running the tests? Try something like `serverJS/testOnly *CORSSuite`?

Reply:
Yes. It's happening on the JVM, too.  I'm trying to minimize it.  It's not just CORS, either.

Output of `sbt 'tests/testOnly *Method*'` on `main`:

```
org.http4s.MethodSuite:
  + Hash[Method]: hash.antisymmetry eq 0.09s
  + Hash[Method]: hash.hash compatibility 0.01s
  + Hash[Method]: hash.reflexivity eq 0.005s
  + Hash[Method]: hash.same as scala hashing 0.007s
  + Hash[Method]: hash.same as universal hash 0.006s
  + Hash[Method]: hash.symmetry eq 0.003s
...
[info] Passed: Total 32, Failed 0, Errors 0, Passed 32
[success] Total time: 4 s, completed Aug 30, 2021 4:55:20 PM
```

Output on this branch:

```
org.http4s.MethodSuite:
[info] Passed: Total 32, Failed 0, Errors 0, Passed 32
[success] Total time: 49 s, completed Aug 30, 2021 4:57:33 PM
```

Reply:
munit got upgraded to 0.7.28.  It's the [buffered logging](https://github.com/scalameta/munit/releases/tag/v0.7.28).

Reply:
Ahhhhh. No wonder test output have been so weird lately.

Reply:
Are you seeing it in other projects?  I opened https://github.com/scalameta/munit/issues/408.  I haven't had time to try it on a smaller project.

Reply:
The buffering is very noticeable for large test suites, that's where I've noticed it most. Also, I've seen disappearing test output sometimes, but that pre-dates this change and only occurs in CI that runs both JVM and JS suites simultaneously.